docs/examples/traefik-ingress.md

```diff
@@ -1,24 +1,25 @@
-# Installing the Traefik Ingress Controller on k0s
+# Install the Traefik Ingress Controller on k0s
 
-In this tutorial, you'll learn how to configure k0s with the
-[Traefik ingress controller](https://doc.traefik.io/traefik/providers/kubernetes-ingress/),
-a [MetalLB service loadbalancer](https://metallb.universe.tf/),
-and deploy the Traefik Dashboard along with a service example.
-Utilizing the extensible bootstrapping functionality with Helm, 
-it's as simple as adding the right extensions to the `k0s.yaml` file 
-when configuring your cluster.
+You can configure k0s with the [Traefik ingress
+controller](https://doc.traefik.io/traefik/providers/kubernetes-ingress/), a
+[MetalLB service loadbalancer](https://metallb.universe.tf/),
+and deploy the Traefik Dashboard using a service sample. With the extensible
+bootstrapping functionality that Helm provides, this is as simple as adding the
+several extensions to the `k0s.yaml` file  when you configure your cluster.
```

Comment:
"as simple as adding the several extensions" sounds wrong.

Reply:
Changed the sentence to read: "To do this you leverage Helm's extensible
bootstrapping functionality to add the correct extensions to the `k0s.yaml` file during cluster configuration."